mod/item.php

```diff
@@ -175,6 +175,19 @@ function item_post(&$a) {
 	$app         = ((x($_REQUEST,'source'))      ? strip_tags($_REQUEST['source'])  : '');
 	$extid       = ((x($_REQUEST,'extid'))       ? strip_tags($_REQUEST['extid'])  : '');
 
+	// Check for multiple posts with the same message id (when the post was created via API)
+	if (($message_id != '') AND ($profile_uid != 0)) {
```

Comment:
Can we agree on using `&&` and `||` instead of `AND` and `OR` in PHP conditions to distinguish them visually from SQL statements?